changelog.md

```diff
@@ -7,6 +7,7 @@
 - [#3835](https://github.com/ignite/cli/pull/3835) Add `--minimal` flag to `scaffold chain` to scaffold a chain with the least amount of sdk modules
 
 ## [`v28.1.1`](https://github.com/ignite/cli/releases/tag/v28.1.1)
+- [#3899](https://github.com/ignite/cli/pull/3899) Introduce plugin.Execute function
```

Comment:
The entry got mixed with the new release, it should be moved to "Unreleased"

Reply:
Good catch, fixed here 2ed60ebaf5aa617ddb9bbbbfc6e7401587231dac